lib/stsci/tools/tests/test_bitmask.py

```diff
@@ -0,0 +1,186 @@
+"""
+A module containing unit tests for the `bitmask` modue.
+
+:Authors: Mihai Cara (contact: help@stsci.edu)
+
+:License: `<http://www.stsci.edu/resources/software_hardware/pyraf/LICENSE>`_
+
+"""
+from __future__ import (absolute_import, division, unicode_literals,
+                        print_function)
+
+import warnings
+import numpy as np
+import pytest
+
+from stsci.tools import bitmask
```

Comment:
Relative import would ensure that the `bitmask` being tested is relative to the path of this test, which might come in handy if there are multiple installations of `stsci.tools` in your environment. However, @drdavella has argued otherwise, so I'll see if he wants to comment here.